src/nIsForName.js

```diff
@@ -22,7 +22,21 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  const empty = [];
+
+  if (names.length === 0) {
+    return empty;
+  }
```

Comment:
Зачем записывать пустой массив в переменную?
```suggestion
  if (!names.length) {
    return [];
  }
```